tests/data/vcf/test.vcf

```diff
@@ -0,0 +1,24 @@
+##fileformat=VCFv4.2
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
```

Comment:
Maybe we need to add a `.gz` version of this VCF for the tests too?

Reply:
I am calling both `bcftools-bgzip` and `bcftools-tabix` each time I need any of those files, but actually I was wondering whether it was the best option, what would be your take on this? Keep the test data as small of possible or get rid of this kind of module dependencies on the tests?

Reply:
I think it would be a good idea to have separate test data for this. Given that it is so small it shouldn't have much of an effect when cloning the repo. Would prefer that the tests are only testing any one module at a time otherwise things will become trickier to maintain if upstream modules are changed.